Classes/Http/Middleware/OpcacheResetHandler.php

```diff
@@ -29,7 +29,9 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
     {
         $action = new OpcacheAction(OpcacheAction::RESET);
 
-        if (strtolower($request->getMethod()) !== $action->getRequestMethod() || $request->getUri()->getPath() !== $action->getUriPath()) {
+        $resetPathDetected = substr_compare($request->getUri()->getPath(), $action->getUriPath(), -(strlen($action->getUriPath())), strlen($action->getUriPath())) === 0;
```

Comment:
This is impossible to read and understand. ;-) Maybe use `str_(starts|ends)_with()`?

Also currently it sounds like we stop processing if "our" path is detected which is contrary to what actually happens.

Reply:
Sorry for pushing this wip-code to the PR.

Well str_ends_with() makes it incompatible with  PHP 7.4 ...
For making it readable I introduces a speaking variable. But I'm happy about a better solution :-)

Reply:
I wouldn't mind requiring `symfony/polyfill-php80` for this.